pratikghimire/WeatherApp
# [Feature] Weather forecast data for Sydney and Melbourne

**Summary/Motivation:**
In this feature, the user can view the weather information for two of the cities, **Sydney** and **Melbourne**. The user can view the current weather information, along with the forecasted weather for the next 6 days. The user has access to multitude of data such as wind speed, wind direction, sunrise, sunset time, temperatures etc.

*NOTE*: The app is using the yahoo weather API to fetch the weather information.

**Testing Strategy:**
*Happy Path*
- Select one of the cities and it should render the weather data for the current day and further 6 days

*Sad Path*
- Try to access the data for invalid id, it should redirect to the home page and show a message

**Security Checklist:**
- [ ] Injection(sql/command)
- [ ] Insecure Direct Object Reference
- [ ] Missing Function Level Access Control
- [ ] Unvalidated Redirects and Forwards (redirect_to)
- [ ] Cross Site Request Forgery(CSRF)
- [ ] Cross Site Scripting(XSS)
- [ ] Unvalidated file/image upload
- [ ] Using Components with Known Vulnerabilities

**Dependencies:**
None

**Screenshots and GIF:**
![screen shot 2018-12-13 at 9 17 01 pm](https://user-images.githubusercontent.com/18133126/49932510-4ca7cd00-ff1d-11e8-9a28-49b97122fe91.png)
